### ISSUE TITLE
Why does clicking submit need us to dedicate our audio to public domain? What does this mean for privacy?

### ISSUE THREAD

**Issue by mario872** [rhasspy/piper-recording-studio#8]
Quick question, I've read through the source code, and can't see any automatic uploading of voice data, but why when clicking submit in the recording studio, a disclaimer says 'By clicking Submit, you agree to dedicate your recorded audio to the public domain (CC0)'. Is the data I record uploaded _automatically_, or is this implying that if I _choose_ to upload my recorded voice data, it should be public domain?
TLDR; Is clicking the submit button uploading my private voice data automatically to a server somewhere?

**Comment by synesthesiam** [comment]
This is only there because I'm hosting Piper Recording Studio at https://contribute.rhasspy.org/ where it is the case that submitted/uploaded audio is donated to the public domain.

But when you run locally, it's up to you what you do with the data. I should add a flag to only show that text for my website :+1:

**Comment by mario872** [comment]
Okay, thank you for clearing that up! (And I must say thank you very much for creating this, it's enabled me to have great fun cloning my voice and getting into the world of machine learning.)
